### PR TITLE
Rename array QueryParam into map to make it work with FOSRestBundle 2…

### PR DESCRIPTION
**Pull request by Bluestart83**
I am targetting this branch, because…

Closes #
Fixes #

## Changelog

```markdown
### Added
- Added some `Class::newMethod` to do great stuff

### Changed

### Deprecated

### Removed

### Fixed

### Security
```

## To do

- [ ] Update the tests
- [ ] Update the documentation
- [ ] Add an upgrade note

## Subject

….0, drop support 1.7 version

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -53,7 +53,7 @@
         "aws/aws-sdk-php": "^2.7",
         "jackalope/jackalope-doctrine-dbal": "^1.1",
         "symfony/phpunit-bridge": "^2.7 || ^3.0",
-        "friendsofsymfony/rest-bundle": "^1.7.4 || ^2.0",
+        "friendsofsymfony/rest-bundle": "^2.0",
```
Comment (Soullivaneuh) [inline composer.json]: We can drop a version support like this. We have to support both.

Actually this is BC break.

**Comment by Soullivaneuh** [comment]
Please first... fill the PR template.

This is not here for decoration. No more review will be done before that.

Regards.

**Comment by Soullivaneuh** [comment]
BTW you created a PR with @desmax commit and **without** any template note respect.

So I will close this one and let @desmax open it if he want to propose and modify his changes.

File: composer.json (hunk 1 above)
Comment (Bluestart83) [inline composer.json]: I understand, so do two different versions Tags. Or fix it differently, but in all case your library is not compatible with 2.0 that you was targetting ;-)

**Comment by Bluestart83** [comment]
Go to Hell! Stop writing crappy Library, or open your mind.

**Comment by greg0ire** [comment]
Wow. Way to speak to a benevolent maintainer of a library he did not even write.
! [Dick move](http://m.imgur.com/gallery/7UOWyiK.gif)

**Comment by Soullivaneuh** [comment]
> open your mind.

Well said! Now think about it.

**Pull request closed without merging**